models/mux_protocol/optimism/mux_protocol_optimism_perpetual_trades.sql

```diff
@@ -0,0 +1,148 @@
+{{ config(
+    
+    alias = 'perpetual_trades',
+    partition_by = ['block_month'],
+    materialized = 'incremental',
+    file_format = 'delta',
+    incremental_strategy = 'merge',
+    unique_key = ['block_date', 'blockchain', 'project', 'version', 'tx_hash', 'evt_index'],
+    post_hook='{{ expose_spells(\'["optimism"]\',
```

Comment:
can we remove this post hook?
moving forward, we'd like to avoid this on spells which feed into final downstream spells